src/systems/systems/src/pneumatic/mod.rs

```diff
@@ -406,6 +457,44 @@ impl DefaultConsumer {
     }
 }
 
+// WING ANTI ICE BLOCK
+pub struct WingAntiIcePushButton {
```

Comment:
I comment here regarding `PNEU_WING_ANTI_ICE_SYSTEM_ON` - I am not sure it should be removed in favor of `..._PB_IS_ON`. `PNEU_WING_ANTI_ICE_SYSTEM_ON` is not just for the push button, it is also for the ECAM text when the WAI is on. So this simvar indicates that the WAI system is working, rather than just light on the pushbutton led. 
I agree that this struct is redundant, and can be replaced by `OnOffFaultPushButton`, however I think that the governing simvar name should not be changed, as it is not related only to the push button.

What do you think?

Reply:
Ok, I see your point. I think we could possibly use two simvars in that case. One (or two if you count the fault) describing only the pushbutton and one describing the system. This can also be used by the ECAM.